tools-core/src/main/java/com/hedera/hashgraph/client/core/remote/MetadataFile.java

```diff
@@ -125,6 +125,6 @@ public boolean equals(final Object o) {
 
 	@Override
 	public int hashCode() {
-		return super.hashCode() + metadataActions.hashCode();
+		return super.hashCode();
```

Comment:
you can remove override if your only calling super. Why remove metadataActions from hashcode?

Reply:
(this comment applies in all the other places where you changed hashCode()

Reply:
I removed the metadata action, because I changed to getting the hashcode of the file contents. Since all of these are files, no further action is really required. Regarding the override, I left it because, in most cases I overrode the `equals` method. If I do one and not the other, SonarCloud complains

Reply:
you can remove both the equals and hashcode overrides in some of the cases.

Reply:
That is correct. Let me go over them and remove them